### PR TITLE
utils: Add support for additional user-agent

### PR DESCRIPTION
**Pull request by Prnyself**

### CODE REVIEW

File: generated.go [hunk 1]
```diff
@@ -23,10 +23,21 @@ const Type = "qingstor"
 
 // Service available pairs.
 const (
+	// AdditionalUserAgent
+	pairAdditionalUserAgent = "qingstor_additional_user_agent"
```
Comment (Xuanwo) [inline generated.go:27]: Maybe we can add `user_agent` in global?

Comment (Prnyself) [reply generated.go:27]: You mean in move this pair into go-storage?

Comment (Xuanwo) [reply generated.go:27]: Yes, other service may also need this pair support?

**Comment by Prnyself** [comment]
Added in [go-storage](https://github.com/aos-dev/go-storage/pull/477) as a global pair

**Pull request closed without merging**